### DISCUSSION TITLE
pulumi-artifactory: deprecation of PermissionTarget and provider for JFrog Platform

### DISCUSSION THREAD

**Discussion by skycaptain** [pulumi/pulumi#16940] [Pulumi Providers] [answered]
With Jfrog revamping the [artifactory](https://registry.terraform.io/providers/jfrog/artifactory) provider, more features (especially access and permission-related ones) are being shifted to the [platform](https://registry.terraform.io/providers/jfrog/platform) provider. For instance, in the latest version, `permission_target` has been deprecated in favor of the [`platform_permission`](https://registry.terraform.io/providers/jfrog/platform/latest/docs/resources/permission) resource. Also, to manage JFrog projects effectively, the [project](https://registry.terraform.io/providers/jfrog/project) provider is now required. It would be fantastic to have a Pulumi provider to fully manage projects on the JFrog Platform. I've kicked things off with [skycaptain/pulumi-jfrog-project](https://github.com/skycaptain/pulumi-jfrog-project), but I don't have the expertise to maintain it professionally. Do you think we could make this provider official and get some support for maintaining it?

**Comment by justinvp** [comment, marked as answer]
Now tracking this at https://github.com/pulumi/pulumi/issues/16942.

**Reply by skycaptain** [reply]
Thanks for the hint. The description of https://github.com/pulumi/pulumi/issues/16942 only mentions the [platform](https://registry.terraform.io/providers/jfrog/platform) provider. Should we start another issue for the [project](https://registry.terraform.io/providers/jfrog/project) provider?

**Reply by justinvp** [reply]
> Should we start another issue for the [project](https://registry.terraform.io/providers/jfrog/project) provider?

Sure. Can you open one?

**Reply by skycaptain** [reply]
https://github.com/pulumi/pulumi/issues/16945
